verilog/tools/ls/symbol-table-handler_test.cc

```diff
@@ -0,0 +1,221 @@
+// Copyright 2017-2023 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "verilog/tools/ls/symbol-table-handler.h"
+
+#include <memory>
+
+#include "absl/strings/string_view.h"
+#include "common/util/file_util.h"
+#include "gtest/gtest.h"
+#include "verilog/analysis/verilog_project.h"
+
+namespace verilog {
+namespace {
+
+constexpr absl::string_view  //
+    kSampleModuleA(
+        "module a;\n"
+        "  assign var1 = 1'b0;\n"
+        "  assign var2 = var1 | 1'b1;\n"
+        "endmodule\n"),
+    kSampleModuleB(
```

Comment:
Avoid using multiple variables in one declaration, split them up. In particular here, where things go over multiple lines and it is hard to even see at a glance that these are two.

```c++
// Instead of
absl::string_view a, b;

// ... make it
absl::string_view a;
absl::string_view b;
```